hardhat.config.ts

```diff
@@ -35,14 +35,12 @@ const config: HardhatUserConfig = {
     regtest: {
       url: getRskNodeUrl(),
       chainId: 33,
-      gas: 6300000,
```

Comment:
based on our conversation, we should not remove this.

Reply:
As I understand it was the opposite: we should delete this. Let's talk about it later.